ikenna-portfolio/portfolio/src/main/webapp/style.css

```diff
@@ -38,6 +38,8 @@
 
 .desc {
   font-size: 16px;
+  margin: auto auto auto auto;
```

Comment:
nit: I think this is the same thing as "margin: auto"

Reply:
You are correct, change is done in new branch for the next pull request